.vtex/buildspec.yml

```diff
@@ -0,0 +1,21 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - echo Installing Package...
```

Comment:
Just to be annoying `echo Installing Package…`

Reply:
Q não entendi

Reply:
`&hellip;` / `…` em vez de três pontos.

Reply:
isso é uma parada que aparece no shell, vou fazer não